src/libYARP_OS/src/NameConfig.cpp

```diff
@@ -319,10 +319,10 @@ ConstString NameConfig::getHostName(bool prefer_loopback, ConstString seed) {
         }
 #ifdef YARP_HAS_ACE
         delete[] ips;
-#else
-        freeifaddrs(ifaddr);
-#endif
```

Comment:
Thanks .. fixed in https://github.com/robotology/yarp/pull/844/commits/fd302953c8206b43d6facc24c3e7f975ad0c7227